### PR TITLE
Setup project and first classes

### PR DESCRIPTION
**Pull request by Sirvelasque**
Add classes
- Person class
- Student class (inherits from person)
- Teacher class (inherits from person)

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  attr_accessor :age, :name
+
+  def initialize(age, name = 'Unknown', parent_permision: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permision = parent_permision
+  end
+
+  private
```
Comment (wuletawwonte) [inline person.rb:16]: - [x] Every instance method after private becomes a private method. So please move the can_use_services method above the private keyword.

File: student.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize()
+    super()
+    @classroom = classroom
+  end
```
Comment (wuletawwonte) [inline student.rb]: - [x] The constructor should have all the parameters of the parent class and with `classroom` as an extra parameter. Please add all parameters in the initialize method and add parameters of the super class in the super method as well.

File: teacher.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
+    @specialization = specialization
+  end
```
Comment (wuletawwonte) [inline teacher.rb]: - [x] The constructor should have all the parameters of the parent class and with specialization as an extra parameter. Please add all parameters in the initialize method and add parameters of the super class in the super method as well.

**Review by wuletawwonte — CHANGES_REQUESTED** [review]
# Hi @Sirvelasque  👋🏿👋🏿👋🏿  

Good job so far!
There are some issues that you still need to work on to go to the next project! You have only a few more steps and you are done! 💪 💪 💪 

giphy

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by t-yanick — APPROVED** [review]
Hi @Sirvelasque, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**